bdk-jvm/lib/src/test/kotlin/org/bitcoindevkit/OfflinePersistenceTest.kt

```diff
@@ -40,4 +40,31 @@ class OfflinePersistenceTest {
             actual = addressInfo.address.toString(),
         )
     }
+
+    @Test
+    fun testPersistenceWithDescriptor() {
+        val connection = Connection(persistenceFilePath)
+
+        val descriptorPub = Descriptor(
```

Comment:
possible TODO if this is the sort of test we want:
- rename function
- rename `val`
- make `val` private